web/packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -119,7 +119,7 @@ export function DesktopSession(props: State) {
 
   if (errorDialog.open) {
     return (
-      <Session {...props} initTdpCli={false} displayCanvas={false}>
+      <Session {...props} connectTdpCli={false} displayCanvas={false}>
```

Comment:
Is `Cli` supposed to be shorthand for "client" here? I can't help but think "command line interface" when I read it.

Would `connect` work just as well?